src/Components/ActivationKey/ActivationKey.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { withRouter, useParams } from 'react-router-dom';
+import pckg from '../../../package.json';
+import Breadcrumbs from '../shared/breadcrumbs';
+const { routes: paths } = pckg;
+import {
+  Text,
+  TextContent,
+  TextVariants,
+  Grid,
+  GridItem,
+  Gallery,
+  GalleryItem,
+} from '@patternfly/react-core';
+import { Main } from '@redhat-cloud-services/frontend-components/Main';
+import {
+  PageHeader,
+  PageHeaderTitle,
+} from '@redhat-cloud-services/frontend-components/PageHeader';
+import AdditionalRepositoriesCard from './AdditionalRepositoriesCard';
+import useActivationKey from '../../hooks/useActivationKey';
+import Loading from '../LoadingState/Loading';
+import SystemPurposeCard from './SystemPurposeCard';
+import WorkloadCard from './WorkloadCard';
+import NoAccessView from '../ActivationKeys/no-access';
+import { useQueryClient } from 'react-query';
+
+const ActivationKey = () => {
+  const queryClient = useQueryClient();
+  const user = queryClient.getQueryData('user');
+  const { id } = useParams();
+  const breadcrumbs = [
```

Comment:
How about moving all breadcrumbs into a shared constant file for better visibility?